temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
Avoid using `pdb` in production code. If this is for debugging purposes, ensure it is removed before committing.